README.rst

```diff
@@ -22,6 +22,9 @@ Simple Example
    # find any entry by its title
    >>> entry = kp.find_entries_by_title('facebook', first=True)
 
+   # find entry by multiple arguments
+   >>> entry = kp.find_entries_by(Title='gmail', UserName='myusername', Password='myPassw0rdXX', first=True)
```

Comment:
I'd be better IMHO to lowercase the kwargs here. This means we need to maintain a dict that translates between kwargs and KeePass XML property names, but makes usage so much easier.